go.mod

```diff
@@ -32,3 +32,5 @@ require (
 	sigs.k8s.io/release-utils v0.3.0
 	sigs.k8s.io/zeitgeist v0.3.0
 )
+
+replace github.com/aquasecurity/libbpfgo => github.com/saschagrunert/libbpfgo v0.2.1-libbpf-0.4.0.0.20211026092140-9b286c168b65
```

Comment:
I think we don’t need the fork. If every file is excluded on our side, then go will not require to build the dependency at all.

Reply:
Thanks.